.gitignore

```diff
@@ -0,0 +1,2 @@
+/config/database.yml
+readme.md
```

Comment:
Do not add readme.md in git ignore. This file should explain information about what this project and how to new developer can setup it.